src/timePicker/TimePicker.tsx

```diff
@@ -36,6 +36,7 @@ export interface TimePickerProps {
   readonly onChange?: (obj: OnChangeParameters) => void
   readonly timeStart?: string
   readonly focus?: boolean
+  readonly filterEmbedded?: boolean
```

Comment:
Rename to `small: boolean` to make it more generic

Reply:
I thought `small` was too generic so I renamed it just before my commit :)
If it's ok for you I'll change it, thanks